### DISCUSSION TITLE
Error with pptx

### DISCUSSION THREAD

**Discussion by pedroaugustoramos** [otahina/PowerPoint-Generator-Python-Project#11] [General]
Hi, there. I'm trying to run this app, but I'm encountering the error: pptx.exc.PackageNotFoundError: Package not found at 'static/presentations/simple.pptx'. Any ideas to help, please? Thank you!

**Comment by pedroaugustoramos** [comment]
Issue was resolved. Please see the Issues section. Thank you

**Comment by akash-mishra2323** [comment]
my api is not working
